docs/quickstart.rst

```diff
@@ -11,6 +11,15 @@ Install a package from `PyPI`_:
     [...]
     Successfully installed SomePackage
 
+Install a distribution already downloaded from `PyPI`_ or got elsewhere.
```

Comment:
*while* the rest of the page is using the term `package`, I think this bit should - s/distribution/package. Be consistent first, if we think it's incorrect enough, then correct it later (i.e separate PR).